### PR TITLE
add marathon cli integration

### PR DESCRIPTION
**Pull request by tamarrow**
After this is merged, we can have it run on teamcity analogously to bin/configure-tests.sh

### CODE REVIEW

File: bin/configure-cli-tests.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+/etc/init.d/zookeeper start
```
Comment (aquamatthias) [inline bin/configure-cli-tests.sh]: do you have to wait for zookeeper to start up?

File: bin/configure-cli-tests.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+cd /marathon
+screen -m -d -S testing -L sbt 'run --internal_store_backend mem --master local'
```
Comment (aquamatthias) [inline bin/configure-cli-tests.sh]: You use the mem storage backend: Intention? To make more real world integration tests, I would omit that parameter and use the default (zk).

File: bin/configure-cli-tests.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+i=0
+while ! grep "Scheduler actor ready" screenlog.0
```
Comment (aquamatthias) [inline bin/configure-cli-tests.sh]: nifty!

File: bin/configure-cli-tests.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (aquamatthias) [inline bin/configure-cli-tests.sh]: That file lives in <marathon_dir>/bin but really is not intended for the user to run.
Will discuss this with you directly.

File: bin/run-cli-tests.sh [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+
+
+# be verbose and print all commands, better for debugging
+# set -x
+
+PROJECT_DIR=`dirname $0`/..
+PROJECT_DIR=$(cd "$PROJECT_DIR"; pwd)
+
+if [ -r "$PROJECT_DIR/bin/run-tests-config.sh" ]; then
```
Comment (aquamatthias) [inline bin/run-cli-tests.sh]: I like, that you reuse the settings mechanism. thanks.

**Comment by aquamatthias** [comment]
Thanks @tamarrow 
- Added some comments, but generally this looks very promising
- I ran the tests locally on my machine and got a lot of failures: `tests/integrations/test_marathon.py ..............FFFFFF.........F.FF..F..FF.F..F..F...........F.` any idea?
- Can you prepare a separate TeamCity job for those tests?

**Comment by tamarrow** [comment]
thanks @aquamatthias! Updated the pr, and I'll add the teamcity build once we fix the cli tests :)

File: Dockerfile.build-cli-base [hunk 6]
```diff
@@ -0,0 +1,32 @@
+FROM java:8-jdk
```
Comment (aquamatthias) [inline Dockerfile.build-cli-base]: can you please create `integrations/dcos-cli/` and move that file here?

File: bin/configure-cli-tests.sh (hunk 4 above)
Comment (aquamatthias) [inline bin/configure-cli-tests.sh]: can you please create `integrations/dcos-cli/` and move that file here?

File: Dockerfile.build-cli-base (hunk 6 above)
Comment (tamarrow) [inline Dockerfile.build-cli-base]: already moved :)

**Comment by aquamatthias** [comment]
Hey @tamarrow this PR is open for a long time. I will close it for now. Please reopen if it is ready.

**Pull request closed without merging**